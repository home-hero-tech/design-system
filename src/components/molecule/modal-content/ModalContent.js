import React from 'react';
import PropTypes from 'prop-types';
import css from '../../organism/modal/Modal.module.scss';

const ModalContent = ({ children }) => (
  <main className={css['c-modal__content']}>{children}</main>
);

ModalContent.propTypes = {
  children: PropTypes.oneOfType([PropTypes.array, PropTypes.object]).isRequired
};

ModalContent.defaultProps = {};

ModalContent.displayName = 'ModalContent';

export default ModalContent;
