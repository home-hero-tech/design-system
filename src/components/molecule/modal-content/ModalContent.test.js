import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ModalContent from './ModalContent';

describe('ModalContent', () => {
  it('should match with the snapshot', () => {
    const children = <div>Content</div>;
    const wrapper = shallow(<ModalContent>{children}</ModalContent>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
