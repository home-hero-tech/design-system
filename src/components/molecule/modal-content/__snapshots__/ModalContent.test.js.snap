// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalContent should match with the snapshot 1`] = `
<main
  className="c-modal__content"
>
  <div>
    Content
  </div>
</main>
`;
